### PR TITLE
doc/mgr: remove diskprediction cloud section

### PR DESCRIPTION
**Pull request by zdover23**
This service is no longer provided, so this
section has to go.

Fixes: https://tracker.ceph.com/issues/46863
Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by zdover23** [comment]
jenkins render docs

File: doc/mgr/diskprediction.rst [hunk 1]
```diff
@@ -50,41 +50,6 @@ Run the following command to use local predictor predict device life expectancy.
     ceph device predict-life-expectancy <device id>
 
 
-Cloud Mode
```
Comment (jdurgin) [inline doc/mgr/diskprediction.rst:53]: there are a few cloud references earlier on this page as well

**Comment by tchaikov** [comment]
@hsiang41 hi Rick, do you happen to know why https://www.diskprophet.com/ is not accessible anymore? is the service offered by api.diskprophet.com still available? if not, do we have any alternative?

**Comment by hsiang41** [comment]
HI Kefu:

The https://www.diskprophet.com is not accessible anymore. You can use local mode disk healthy prediction.

**Comment by zdover23** [comment]
@jdurgin Can you weigh in on this?

**Comment by tchaikov** [comment]
@zdover23 @jdurgin please see #36680

**Comment by tchaikov** [comment]
@hsiang41 ack. i am removing the cloud-based failure prediction module in #36680

**Comment by jdurgin** [comment]
looks like kefu's change is a superset of this, so closing this one

**Pull request closed without merging**